frontend/lib/src/components/shared/BaseButton/DynamicButtonLabel.tsx

```diff
@@ -24,12 +24,14 @@ export interface DynamicButtonLabelProps {
   icon?: string
   label?: string
   iconSize?: IconSize
+  useSmallerFont?: boolean
 }
 
 export const DynamicButtonLabel = ({
   icon,
   label,
   iconSize,
+  useSmallerFont = false,
```

Comment:
nit: Should we use the same terminology as in `StreamlitMarkdown` where we use `largerLabel`?

Reply:
Hmm, I choose it this way since almost all buttons use the larger label. The only exception is pills/segmented control.

Reply:
couldn't you do `largerLabel = true` in this case to address your concern?

Reply:
Yeah that is a good pattern too. I don't have a strong opinion and will leave it to you to decide whether keeping the argument names consistent across the components outweighs that preference or not.

Reply:
ok, I think I'm keeping it as is since we are also using useSmallerFont inside StreamlitMarkdown for the same